lib/cops/anchor/avoid_implicit_super.rb

```diff
@@ -0,0 +1,21 @@
+module Anchor
+  class AvoidImplicitSuper < RuboCop::Cop::Cop
+    MSG = "Avoid implicit `super` calls. Prefer `super(**kwargs)`".freeze
```

Comment:
I wonder if it's helpful to offer further options:

```suggestion
    MSG = "Avoid implicit `super` calls. Prefer `super(**kwargs)`, `super()`," \
      "or a mix of explicit/implicit arguments, e.g. `super(id:, title:, **kwargs)`"
      .freeze
```